src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -46,8 +46,10 @@ class SymfonyStyle extends OutputStyle
     public function __construct(InputInterface $input, OutputInterface $output)
     {
         $this->input = $input;
-        $this->lineLength = min($this->getTerminalWidth(), self::MAX_LINE_LENGTH);
         $this->bufferedOutput = new BufferedOutput($output->getVerbosity(), false, clone $output->getFormatter());
+        // Windows cmd wraps lines as soon as the terminal width is reached, whether there are following chars or not.
+        // https://github.com/symfony/symfony/pull/14740
+        $this->lineLength = min($this->getTerminalWidth() - (int) (DIRECTORY_SEPARATOR === '\\'), self::MAX_LINE_LENGTH);
```

Comment:
Can you remove the github reference, there is no need for that as the merge commit already contains everything we need to know.